ui/lib/core/addon/components/info-table-item-array.hbs

```diff
@@ -0,0 +1,59 @@
+<div data-test-info-table-item-array {{did-insert (perform this.fetchOptions)}}>
+  {{#if @isLink}}
+    <div data-test-row-value={{@label}}>
+      {{#each this.displayArrayAmended as |name|}}
```

Comment:
does this fail gracefully when `this.displayArrayAmended = null` ?

Reply:
yep, good question. I double checked.